cpp/test/neighbors/ann_cagra.cuh

```diff
@@ -15,6 +15,8 @@
  */
 #pragma once
 
+#undef RAFT_EXPLICIT_INSTANTIATE_ONLY  // Search with filter instantiation
```

Comment:
This leads to 3x build time for the CI (30 mins -> 90 mins)

Reply:
The CI build time is actually still around the 30 minutes with the instantiation of the search kernels with a test pre-filter.